core/templates/components/common-layout-directives/navigation-bars/side-navigation-bar.component.html

```diff
@@ -247,7 +247,7 @@
                  alt="learn-math-logo">
           </picture>
         </div>
-        <a href="/learn/{{DEFAULT_CLASSROOM_URL_FRAGMENT}}" smartRouterLink="/learn/{{DEFAULT_CLASSROOM_URL_FRAGMENT}}" class="e2e-mobile-test-mathematics-link mb-1">
+        <a href="/learn/{{DEFAULT_CLASSROOM_URL_FRAGMENT}}" tabindex="-1" smartRouterLink="/learn/{{DEFAULT_CLASSROOM_URL_FRAGMENT}}" class="e2e-mobile-test-mathematics-link mb-1">
```

Comment:
Can you please add a comment above it to explain why the tab index is negative, so that any developer in future does not change it.

Reply:
@Rijuta-s Done